README.md

```diff
@@ -24,7 +24,44 @@ __Note:__ On Linux and MacOS, you will need to enable executable permission befo
 
 _e.g._ `$ chmod u+x fpm-v0.1.0-linux-x86_64`
 
-### Build from source
+For other platforms and architectures have a look at the [bootstrapping instructions](#bootstrapping-fpm).
```

Comment:
Tried this link out at <https://github.com/awvwgk/fortran-package-manager/tree/readme#bootstrapping-fpm> and it doesn't work for me on Chrome, even when changed to `#bootstrapping-instructions`. Presumably this is because of the `<details>`.

Reply:
This was just me mistyping the link reference, should be fixed now.